Tests/Provider/AbstractProviderTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Provider;
+
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\MediaBundle\Provider\MediaProviderInterface;
+use Sonata\MediaBundle\Tests\Form\Type\AssertFormReferenceCompatibleWithSymfony28Trait;
+use Symfony\Component\Form\FormBuilder;
+use Symfony\Component\Form\FormTypeInterface;
+
+/**
+ * Class AbstractTypeTest.
+ *
+ * @author Virgile Vivier <virgilevivier@gmail.com>
+ */
+abstract class AbstractProviderTest extends \PHPUnit_Framework_TestCase
```

Comment:
@core23 Not sure to understand why do you mean by that. Replace `AbstractProviderTest` to `AbstractProviderTestCase` ?
If yes, I didn't find any file with this suffix in this project.